tests/api_tests/nodes/test_views.py

```diff
@@ -0,0 +1,638 @@
+# -*- coding: utf-8 -*-
+import mock
+from nose.tools import *  # flake8: noqa
+
+from framework.auth.core import Auth
+from website.models import Node
+from tests.base import ApiTestCase, fake
+from tests.factories import UserFactory, ProjectFactory, FolderFactory, DashboardFactory, NodeFactory, PointerFactory
+
+
+class TestNodeList(ApiTestCase):
```

Comment:
The opposite of that is happening. api_v2_url_for is disappearing and we will use string-formatted URLs. For a public API, the URLs are the canonical reference, while the methods behind them could changes three times a day and not affect the important thing. I've pulled them out of the tests, but I do need to go through the actual code and remove all references to api_v2_url_for (and to delete the helper function).